pkgs/by-name/ha/harbor-cli/package.nix

```
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  testers,
+  harbor-cli,
+  installShellFiles,
+}:
+
+buildGoModule rec {
+  pname = "harbor-cli";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "goharbor";
+    repo = "harbor-cli";
+    rev = "v${version}";
+    hash = "sha256-WSADuhr6p8N0Oh1xIG7yItM6t0EWUiAkzNbdKsSc4WA=";
+  };
+
+  vendorHash = "sha256-UUD9/5+McR1t5oO4/6TSScT7hhSKM0OpBf94LVQG1Pw=";
+
+  nativeBuildInputs = [ installShellFiles ];
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X github.com/goharbor/harbor-cli/cmd/harbor/internal/version.Version=${version}"
+  ];
+
+  postInstall = ''
```

See https://github.com/NixOS/nixpkgs/issues/308283. This is counterintuitive tho
```suggestion
  postInstall = lib.optionalString (stdenv.buildPlatform.canExecute stdenv.hostPlatform) ''
```